<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>297</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Randomize options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="oscGroupChk">
        <property name="text">
         <string>Oscillators</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="osc2">
        <property name="text">
         <string>Osc 2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">oscGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="osc3">
        <property name="text">
         <string>Osc 3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">oscGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="osc1">
        <property name="text">
         <string>Osc 1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">oscGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QCheckBox" name="lfo1">
        <property name="text">
         <string>LFO 1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lfoGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="lfo3">
        <property name="text">
         <string>LFO 3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lfoGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="lfo2">
        <property name="text">
         <string>LFO 2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lfoGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="lfoGroupChk">
        <property name="text">
         <string>LFOs</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="QCheckBox" name="env3">
        <property name="text">
         <string>Envelope 3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">envGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="env4">
        <property name="text">
         <string>Envelope 4</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">envGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="ampEnv">
        <property name="text">
         <string>Amplifier envelope</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">envGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="filterEnv">
        <property name="text">
         <string>Filter envelope</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">envGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="envGroupChk">
        <property name="text">
         <string>Envelopes</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCheckBox" name="filter1">
        <property name="text">
         <string>Filter 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="filter2">
        <property name="text">
         <string>Filter 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="arpUser">
        <property name="text">
         <string>Arp. user pattern</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="effect1">
        <property name="text">
         <string>Effect 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="effect2">
        <property name="text">
         <string>Effect 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="arpeggiator">
        <property name="text">
         <string>Arpeggiator</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="mixer">
        <property name="text">
         <string>Mixer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="amplifier">
        <property name="text">
         <string>Amplifier</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="commons">
        <property name="text">
         <string>Commons</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="modMatrix">
        <property name="text">
         <string>Modulation matrix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="oscGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="lfoGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="envGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
